doc/user/installing-graalvm.md

```diff
@@ -12,10 +12,10 @@ for TruffleRuby to run correctly.
 GraalVM is available in a Community Edition, which is open-source, and an
 Enterprise Edition which has better performance and scalability.
 
-The Community Edition is available only for Linux, but is free for production
-use. The Enterprise Edition is available for both macOS and Linux, and is free
+The Community Edition is free for production use. The Enterprise Edition is free
 for evaluation but not production use. Commercial support is available for the
-Enterprise Edition.
+Enterprise Edition. Both editions are available for Linux, macOS and Windows on
+x86 64-bit systems.
```

Comment:
Should we clarify that TruffleRuby does not work on Windows yet here?
Maybe it's clear enough from `System compatibility` in the README.

Reply:
Updated to clarify!